common/src/lib.rs

```diff
@@ -19,6 +20,11 @@ pub struct EventData {
 
 pub const CHANNELS_PER_DIGITIZER: usize = 8;
 
+pub fn init_tracing_subscriber() {
+    let env_filter = EnvFilter::try_from_default_env().unwrap_or_else(|_| EnvFilter::new("info"));
```

Comment:
You do not need this, `tracing_subscriber` will honour the level set by `RUST_LOG` by default.

Reply:
This does not appear to work properly. The `env-filter` feature is included in the main `Cargo.toml` file but calling `tracing_subscriber::fmt().init()` at the beginning of `main` does not read from `RUST_LOG`.

This only seems to work if you use the full:

```rust
tracing_subscriber::fmt()
    .with_env_filter(EnvFilter::from_default_env())
    .init();
```

Reply:
It does work, see https://docs.rs/tracing-subscriber/latest/tracing_subscriber/fmt/fn.init.html, the behaviour is dependant on the feature flags, without `env-filter` it respects `RUST_LOG` in the same way `env_logger` does.